src/transport_ofi.c

```diff
@@ -578,48 +598,65 @@ static inline int atomic_limitations_check(void)
 
     int j;
     /* Binary OPS check */
-    for(i=0; i<3; i++) {//DT
+    for(i=0; i<4; i++) {//DT
```

Comment:
Could you pull out this constant and #define it right next to the array declaration?